src/outfit.ts

```diff
@@ -203,8 +174,11 @@ export class Outfit {
       }
     }
   }
-}
 
-function canChargeVoid(): boolean {
-  return get("_voidFreeFights") < 5 && get("cursedMagnifyingGlassCount") < 13;
+  clone(): this {
+    const result = { ...this };
```

Comment:
I think you need to clone with new Outfit(); I had something similar to this in CombatStrategy but ran into issues. TBH not sure though.

Reply:
I'll go ahead and change it. If I can ever get a linked grimoire working, I'll verify if the easier approach is sufficient.